qa/tasks/cephfs/filesystem.py

```diff
@@ -611,6 +613,15 @@ def create(self):
 
         self.getinfo(refresh = True)
 
+    def rm(self, name='cephfs'):
```

Comment:
@rishabh-d-dave is working on a similar method here:

https://github.com/ceph/ceph/pull/32581/files#diff-b3a56ab86d2e46ef827df2b8b21fc142R585

Suggest you pull this refactor commit out into a separate PR that blocks on the merge of #32581.

Reply:
Yep, makes sense. Done.